src/components/admin/admin-item.js

```diff
@@ -52,17 +31,24 @@ adminApp.AdminItem = {
       }
     };
 
-    this.displayDetailBox = adminApp.ToggleDiv.toggler();
+    displayDetailBox = adminApp.ToggleDiv.toggler();
+
+    return {
```

Comment:
much better! I think we should use this pattern across all components. the `this` pattern was badly inherited from mithril docs, I guess.